packages/client/src/views/OfficeHome/queries.ts

```diff
@@ -11,39 +11,6 @@
  */
 import gql from 'graphql-tag'
 
-const allSearchFields = `
-  id
-    type
-    registration {
-      status
-      contactRelationship
-      contactNumber
-      trackingId
-      eventLocationId
-      registrationNumber
-      registeredLocationId
-      duplicates
-      createdAt
-      modifiedAt
-    }    
-    ... on BirthEventSearchSet {
-      dateOfBirth
-      childName {
-        firstNames
-        familyName
-        use
-      }
-    }
-    ... on DeathEventSearchSet {
-      dateOfDeath
-      deceasedName {
-        firstNames
-        familyName
-        use
-      }
-    }
-  `
-
```

Comment:
Was there any particular reason for removing this? Maybe we should use [Fragments](https://www.apollographql.com/docs/react/data/fragments/#example-usage) instead?

Reply:
Yea, the AST parser of codegen doesn't support that unfortunately and agreed. Fragments might just work. I'll give it a try

Reply:
It worked! Thanks for the suggestion

Reply:
Ok cool